lib/html/header-material.js

```diff
@@ -99,6 +100,23 @@ export default async function run (doc, opt, ctx) {
       metaEls.push(el('dd', {}, [el('a', { href: url.toString() }, [name])]));
     })
   }
+  if (ctx.github) {
+    const { repository, defaultBranch } = ctx.github
```

Comment:
It's probably worth:
1. Erroring if `repository` is not defined and
2. Defaulting `defaultBranch` to `main.